src/project/types/single-file/single-file.ts

```diff
@@ -40,11 +40,12 @@ export async function singleFileProjectContext(
   const projectCacheBaseDir = temp.createDir();
 
   const result: ProjectContext = {
+    clone: () => result,
     resolveBrand: (fileName?: string) => projectResolveBrand(result, fileName),
     dir: normalizePath(dirname(source)),
     engines: [],
     files: {
-      input: [],
+      input: [normalizePath(source)],
```

Comment:
Is this to solve the relative vs absolute path problem? 

Or not directly related? I wonder if it can help 🤔